src/java/org/apache/cassandra/service/accord/AccordJournalValueSerializers.java

```diff
@@ -30,21 +30,27 @@
 import accord.primitives.TxnId;
 import org.apache.cassandra.io.util.DataInputPlus;
 import org.apache.cassandra.io.util.DataOutputPlus;
+import org.apache.cassandra.net.MessagingService;
 import org.apache.cassandra.service.accord.journal.AccordTopologyUpdate;
 import org.apache.cassandra.service.accord.serializers.CommandStoreSerializers;
 import org.apache.cassandra.service.accord.serializers.Version;
 
-import static accord.api.Journal.Load.ALL;
 import static accord.local.CommandStores.RangesForEpoch;
 
 // TODO (required): test with large collection values, and perhaps split out some fields if they have a tendency to grow larger
 // TODO (required): alert on metadata size
 // TODO (required): versioning
 public class AccordJournalValueSerializers
 {
-    public interface FlyweightSerializer<ENTRY, IMAGE>
+    private static final int messagingVersion = MessagingService.VERSION_40;
```

Comment:
why add this back?  This isn't the version accord journal uses

Reply:
its private and don't see anything in this class use it, so its dead code?

Reply:
This looks to be my mistake